etc/HIProdOfflineConfiguration.py

```diff
@@ -1494,15 +1515,22 @@
                aod_to_disk=True,
                write_dqm=True,
                disk_node="T2_US_Vanderbilt",
-               alca_producers=["EcalUncalZElectron", "EcalUncalWElectron", "EcalESAlign", "MuAlCalIsolatedMu", 
-                               "TkAlDiMuonAndVertex", "HcalCalHO", "HcalCalIsoTrkProducerFilter", "HcalCalHBHEMuonProducerFilter",
-                               "SiStripCalZeroBias", "SiStripCalMinBias","TkAlMinBias",
-                               "HcalCalIsolatedBunchSelector", "HcalCalIterativePhiSym","HcalCalMinBias",
-                               "TkAlJpsiMuMu", "TkAlUpsilonMuMu","TkAlZMuMu","TkAlMuonIsolated"],
```

Comment:
mmmh, I think the request was only to remove `"SiStripCalMinBias","TkAlMinBias"` (which are the ones producing large files) - NOT all of them.

Reply:
@saumyaphor4252

Reply:
@LinaresToine Sorry I misunderstood earlier. Can you add these back just removing the `SiStripCalMinBias` and `TkAlMinBias`. Thanks!